CASMCMS-9329: Added cfs components updatemany option (#5841)
Cray-HPE/docs-csm: operations/configuration_management/CFS_Commands_Cheat_Sheet.md

> * (`ncn-mw#`) Clear the state and trigger configuration on a component:
> 
>     > For any of these components with a desired configuration set, this will trigger automatic configuration.

Reviewer: ```suggestion
* (`ncn-mw#`) Clear the state of a component:

    > If this component has a desired configuration set, then this will trigger automatic configuration.
```

    cray cfs v3 components describe <xname> --state-details true
    ```

## Managing many components

* (`ncn-mw#`) Enable or disable automatic configuration for a list of components:

    ```bash
    cray cfs v3 components updatemany --filter-ids <xname1>,<xname2>,... --enabled <true/false>
    ```

* (`ncn-mw#`) Set the desired configuration for a list of components:

    ```bash
    cray cfs v3 components updatemany --filter-ids <xname1>,<xname2>,... --desired-config <example-config>
    ```

> * (`ncn-mw#`) Clear the state and trigger configuration on a list of components:

Reviewer: I realize you were just copying the language used for the single-xname command above. I recommend making these edits to that command as well, because it applies there too.

```suggestion
* (`ncn-mw#`) Clear the state on a list of components:

    > For any of these components with a desired configuration set, this will trigger automatic configuration.
    > See [Automatic Configuration Management](Automatic_Configuration_Management.md) for more details.
```

Author: done

    ```bash
    cray cfs v3 components updatemany --filter-ids <xname1>,<xname2>,... --enabled true --state []
    ```

* (`ncn-mw#`) Enable or disable automatic configuration for a list of components based on status:

    ```bash
    cray cfs v3 components updatemany --filter-status <status> --enabled <true/false>
    ```

* (`ncn-mw#`) Enable or disable automatic configuration for a list of components based on their configuration flag:

    ```bash
    cray cfs v3 components updatemany --filter-enabled <true/false> --enabled <true/false>
    ```

* (`ncn-mw#`) Enable or disable automatic configuration for a list of components based on their tag:

    ```bash
    cray cfs v3 components updatemany --filter-tags <key1=value1>,<key2=value2>,... --enabled <true/false>
    ```

* (`ncn-mw#`) Enable or disable automatic configuration for a list of components based on configuration name:

    ```bash
    cray cfs v3 components updatemany --filter-config-name <configuration name> --enabled <true/false>
    ```

> 

Reviewer: I'd suggest adding at least one example indicating that you can specify multiple filters in the same CLI command, which will operate on components which match all of the specified filters.

Author: added an example of enable/disable configuration for components filterd by ID and configuration flag
